ClientRM/KeyLogger/KeyLogger/DataTransfer.cpp

```diff
@@ -38,8 +41,10 @@ BOOL StartTransfer() {
 
```

Comment:
Crash was happening here when trying to EnterCriticalSection on csLinkList because csLinkList was not initialized at all. It used to get initialized only if a key press occurred in the hooked program.